app/views/posts/show.html.erb

```diff
@@ -1,2 +1,15 @@
-<h1>Post #<%= @post.id %> of  user #<%= @user %></h1>
-<h3>Title: <%= @post.title %>, Text: <%= @post.text %>, Created at: <%= @post.created_at %></h3>
\ No newline at end of file
+<div class="user_bio"
+  <b>Post by <%= @user.name %></b>
+  <p>Comments: <%= @post.comments_counter %>, Likes: <%= @post.likes_counter %> </p>
+  <p>Title: <%= @post.title %></p>
+  <p><%= @post.text %></p>  
+</div>
+
+<div class='div-comments'>
+  <% @post.five_recent_comments(@post.id).each do |comment| %>
+     <div class='comments'>
+        <span style="font-weight :bold font-size: 10px">Username <%= comment.user_id %>:</span> <%= comment.text %>
+      </div>
+  <% end %>        
+</div>
```

Comment:
- [ ] @cmolinan great work on your views, however, the design in your post-show page doesn't match the wireframe given, below is a comparison

Wireframe Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/204989263-4c8ee197-f02a-411c-a5a7-ce5bf64f2d65.png)  |  ![image](https://user-images.githubusercontent.com/61408860/204989390-fa8d1ffd-4f68-416d-b9a0-12b3f2c05f2f.png)

Reply:
@danmainah Done !